saagieapi/projects/projects.py

```diff
@@ -225,6 +224,113 @@ def __get_jobs_for_project(self, jobs_technologies_allowed) -> List:
             )
         return [{"id": t} for t in tech_ids]
 
+    def get_rights(self, project_id: str) -> Dict:
+        """List rights associated for the project
+
+        Parameters
+        ----------
+        project_id : str
+            UUID of your project (see README on how to find it)
+
+        Returns
+        -------
+        dict
+            Dict of rights associated for the project
+        """
+        query = gql(GQL_GET_PROJECT_RIGHTS)
+        return self.client.execute(query, variable_values={"id": project_id})
+
+    def edit(
+        self,
+        project_id: str,
+        name: str = None,
+        group: str = None,
+        role: str = None,
+        description: str = None,
+        jobs_technologies_allowed: Dict = None,
+        apps_technologies_allowed: Dict = None,
+    ) -> Dict:
+        """Edit a project
+
+
+        Parameters
+        ----------
+        project_id : str
+            UUID of your project (see README on how to find it)
+        name : str, optional
+            Name of the project
+            If not filled, defaults to current value, else it will change the job's name
+        group : None or str, optional
+            Authorization management: name of the group to add the given role to
+            If not filled, defaults to current value, else it will change the group
+        role : str, optional
+            Authorization management: role to give to the given group on the project
+            If not filled, defaults to current value, else it will change the role
+        description : str, optional
+            Description of the project
+            If not filled, defaults to current value, else it will change the job's description
+        jobs_technologies_allowed:list, optional
+            Dict of catalog and jobs technologies allowed for the project
+            If not filled, defaults to current value, else it will change the jobs technologies allowed
+        apps_technologies_allowed:list, optional
+            Dict of catalog and apps technologies allowed for the project
+            If not filled, defaults to current value, else it will change the apps technologies allowed
+
+        Returns
+        -------
+        dict
+            Dict of created project
+
+        Raises
+        ------
+        ValueError
+            If given unknown role value
+        """
+        params = {"projectId": project_id}
+        previous_project_version = self.get_info(project_id)["project"]
+        if role:
+            if role == "Manager":
+                role = "ROLE_PROJECT_MANAGER"
+            elif role == "Editor":
+                role = "ROLE_PROJECT_EDITOR"
+            elif role == "Viewer":
+                role = "ROLE_PROJECT_VIEWER"
+            else:
+                raise ValueError("'role' takes value in ('Manager', 'Editor'," " 'Viewer')")
+
+        if group:
```

Comment:
Duplicate code with the create method. Create a function that can be called from the create and edit methods